src/app/modules/item/pages/item-display/item-display.component.html

```diff
@@ -8,6 +8,8 @@
       <alg-error
         *ngIf="initError$ | async; else urlError"
         i18n-message message="It usually occurs when task url is invalid. If the task url is valid and the problem persists, please contact us."
+        [showRefreshButton]="true"
+        (refresh)="refresh.emit()"
       ></alg-error>
```

Comment:
The errors with refresh buttons are only displayed to authenticated users that can edit the task (and it works well by the way :+1: ).
I think we should also display that reload button in the default error message for other users (a bit upper in the code)